### ISSUE TITLE
Sonification for Laser

### ISSUE THREAD

**Issue by samreid** [phetsims/wave-interference#380]
Notes from @Ashton-Morris 

> Light: I have a first sound for the light beam
>  This sound should loop (John has a loop trick that he used in our other sims hopefully that will work here as well). The frequency should be tied to the pitch of this sound and we need to make it stop and restart when the frequency slider is moved. I think the should leave the amplitude modulation for light alone for the time being. When we experiment with the speaker’s amplitude options and come up with a good solution we can import that idea to the Light example

**Comment by samreid** [comment]
> This sound should loop (John has a loop trick that he used in our other sims hopefully that will work here as well)

@jbphet Can you clarify on that part?

**Comment by jbphet** [comment]
Well, I wouldn't call it a trick.  Basically, I ran into problems creating the loop and starting it but leaving the volume at zero and just turning it up when needed.  One problem was that on some browsers it wouldn't start at all because of the autoplay policy.  On others, an initial sound would be heard at startup because the volume wouldn't update until a signal started going through the gain node.  What ended up working was to create the loop but not start it until it was needed, then to also stop it if the volume goes to zero.  This was the approach used for the `chargesInTheBodySoundClip` in `JohnTravoltageView`, and I'd suggest using that as an example.  This could be encapsulated into a sound generator if desired, but it's just inlined in the example that I cited.

Another reason that I took this approach is that it seems to be the spirit of the Web Audio API that sound generators like oscillators and source buffer nodes should be stopped when they are not in use.  I've seen this written down a few times, but never anywhere I would consider very "official".  I think it makes sense, though, since it probably consumes resources of some kind for these sound generators do do their thing.
